### DISCUSSION TITLE
23.10 Mantic Minotaur

### DISCUSSION THREAD

**Discussion by stallion25** [Joshua-Riek/ubuntu-rockchip#663] [General]
I started my rocket today and new upgrade showed up. Worth ?
![mantic minotaur](https://github.com/Joshua-Riek/ubuntu-rockchip/assets/129460841/9845d059-5496-42f2-b9ba-46df5a778b63)

**Comment by JFLim1** [comment]
3D GPU hw acceleration and VPU hw acceleration might no longer work with Mantic Minotaur.

**Comment by Joshua-Riek** [comment]
Do not upgrade it will break your system.

**Reply by stallion25** [reply]
tried on sd card :) it crashed bash package :D
